internal/events/event.go

```diff
@@ -459,29 +465,30 @@ func (e Events) queryEvents(ctx context.Context, params *common.V2GetEventsParam
 
 	params.Limit, params.Offset = preparePaginationParams(params.Limit, params.Offset)
 	if *params.Limit == 0 {
-		return make([]*common.Event, 0), eventSeverityCount, nil
+		return make([]*common.Event, 0), eventSeverityCount, swag.Int64(0), nil
```

Comment:
I don't think we should be returning zero for eventCount here. I think we should return eventCount.

It looks like params.Limit is being used to paginate through results and it seems from the existing code that the significance of `*params.Limit == 0` is to return an empty resultset in the paged data. This does not however change the reality that the `eventCount` has a value that is potentially nonzero.

Better to return `eventCount` here instead of `swag.Int64(0)`

Reply:
Thanks for that, I fixed it